x/axelarcork/types/axelarcork.go

```diff
@@ -6,19 +6,9 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	tmbytes "github.com/tendermint/tendermint/libs/bytes"
 )
 
-func (c *Cork) InvalidationScope() tmbytes.HexBytes {
-	addr := common.HexToAddress(c.TargetContractAddress)
-	return crypto.Keccak256Hash(
-		bytes.Join(
-			[][]byte{addr.Bytes(), c.EncodedContractCall},
-			[]byte{},
-		)).Bytes()
-}
-
-func (c *Cork) IDHash(blockHeight uint64) []byte {
+func (c *AxelarCork) IDHash(blockHeight uint64) []byte {
```

Comment:
On reviewing this again, I think `AxelarCork` IDs need to have the chain ID as one of the hashed values.